card/spec/lib/card/reference_spec.rb

```diff
@@ -65,7 +65,7 @@
   it "pickup new links on rename" do
     @l = newcard("L", "[[Ethan]]")  # no Ethan card yet...
     @e = newcard("Earthman")
-    @e.update_attributes! :name => "Ethan"  # NOW there is an Ethan card
+    @e.update_attributes! name: "Ethan"  # NOW there is an Ethan card
```

Comment:
Unnecessary spacing detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.